libs/ui/switch/helm/src/lib/hlm-switch.component.ts

```diff
@@ -0,0 +1,106 @@
+import { Component, EventEmitter, Input, Output, booleanAttribute, computed, forwardRef, signal } from '@angular/core';
+import { NG_VALUE_ACCESSOR } from '@angular/forms';
+import { hlm } from '@spartan-ng/ui-core';
+import { BrnSwitchComponent, BrnSwitchThumbComponent } from '@spartan-ng/ui-switch-brain';
+import { ClassValue } from 'clsx';
+import { HlmSwitchThumbDirective } from './hlm-switch-thumb.directive';
+
+export const HLM_SWITCH_VALUE_ACCESSOR = {
+	provide: NG_VALUE_ACCESSOR,
+	useExisting: forwardRef(() => HlmSwitchComponent),
+	multi: true,
+};
+
+@Component({
+	selector: 'hlm-switch',
+	imports: [BrnSwitchThumbComponent, BrnSwitchComponent, HlmSwitchThumbDirective],
+	standalone: true,
+	host: {
+		'[attr.class]': 'null',
+		'[attr.id]': 'null',
+		'[attr.aria-label]': 'null',
+		'[attr.aria-labelledby]': 'null',
+		'[attr.aria-describedby]': 'null',
+	},
+	template: `
+		<brn-switch
+			[class]="_computedClass()"
+			[checked]="_checked()"
+			(changed)="handleChange($event)"
+			(touched)="_onTouched()"
+			[id]="id"
+			[aria-label]="ariaLabel"
+			[aria-labelledby]="ariaLabelledby"
+			[aria-describedby]="ariaDescribedby"
+		>
+			<brn-switch-thumb hlm />
+		</brn-switch>
+	`,
+	providers: [HLM_SWITCH_VALUE_ACCESSOR],
+})
+export class HlmSwitchComponent {
+	private readonly _userCls = signal<ClassValue>('');
+	@Input()
+	set class(userCls: ClassValue) {
+		this._userCls.set(userCls);
+	}
+
+	@Output()
+	public changed = new EventEmitter<boolean>();
+
+	handleChange(value: boolean): void {
+		const previousChecked = this._checked();
+		this._checked.set(value);
```

Comment:
this feel a little weird. On the one side we use the previousChecked and ignore the value from the change event on the other side we use the value to set the checked value. Maybe we just use one of them? I know they should be in sync anyways but it just feels off.

Reply:
Yes this is weird, must have been something I left when trying so many different things. 
Need to clean up here.